tests/js/fixtures/sentryAppComponent.js

```diff
@@ -3,19 +3,38 @@ export function SentryAppComponent(params = {}) {
     uuid: 'ed517da4-a324-44c0-aeea-1894cd9923fb',
     type: 'issue-link',
     schema: {
-      link: {
+      create: {
         required_fields: [
-          {type: 'text', name: 'a', label: 'A', default: 'issue.title'},
-          {type: 'textarea', name: 'c', label: 'C', default: 'issue.description'},
+          {
+            type: 'text',
+            name: 'title',
+            label: 'Title',
+            default: 'issue.title',
+          },
+          {
+            type: 'textarea',
+            name: 'description',
+            label: 'Description',
+            default: 'issue.description',
+          },
           {
             type: 'select',
             name: 'numbers',
             label: 'Numbers',
-            options: [['one', 1], ['two', 2]],
+            choices: [[1, 'one'], [2, 'two']],
```

Comment:
I'm confused as to what representation we want here, is it the schema from the user or is it the result of what we get from the `Preparer`?

Reply:
The result of the `Preparer`. This is passed to components, so it should reflect what we receive via `props`, essentially.